### PR TITLE
Progress bar eszter

### PR DESCRIPTION
**Pull request by etterke**
Please check my pull request:
- generated new component for progress bar
- created parent-child relationship with notification
- renamed variables according to changes in backend

thankx

### CODE REVIEW

File: Tribes/src/app/notification/notification.component.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+div{
```
Comment (EggDice) [inline Tribes/src/app/notification/notification.component.css]: missing space after selector

File: Tribes/src/app/notification/notification.component.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    list-style-type: none;
+    color: whitesmoke;
+}
+
+img{
```
Comment (EggDice) [inline Tribes/src/app/notification/notification.component.css]: missing space after selector

File: Tribes/src/app/progress-bar/progress-bar.component.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  constructor() { }
+
+  ngOnInit() {
+    console.log(this.checkTimeDifference(this.notification));
+  }
+
+  checkTimeDifference(notification: Notification): number {
+    const currentTime = new Date().getTime();
+    const timeDiff = (notification.finishedAt - currentTime) / 1000 / 60;
```
Comment (EggDice) [inline Tribes/src/app/progress-bar/progress-bar.component.ts]: If you are storing minutes you could call it that :)

Comment (etterke) [reply Tribes/src/app/progress-bar/progress-bar.component.ts]: renamed variable to minuteDiff and totalMinutes

**Pull request merged**